spec/lib/que/job_spec.rb

```diff
@@ -26,7 +26,7 @@
         queue: "default",
         priority: 100,
         job_class: "Que::Job",
-        retryable: false,
+        retryable: true,
```

Comment:
This proves the point... The default value for `retryable` is [`true`](https://github.com/gocardless/que/blob/97218e24ce27c76152d5c0d713b0ac833cd04193/lib/que/job.rb#L19)